DependencyInjection/Configuration.php

```diff
@@ -54,10 +54,10 @@ public function getConfigTreeBuilder()
                 ->booleanNode('use_flash_notifications')->defaultTrue()->end()
                 ->booleanNode('use_username_form_type')->defaultTrue()->end()
                 ->arrayNode('from_email')
-                    ->addDefaultsIfNotSet()
+                    ->isRequired()
```

Comment:
Indeed, seems like I spoke too fast ... so this means that the top `from_email` node is just a helper node used to propogate its values to the `resetting` and `confirmation` nodes and it's not used by any other part of the bundle ?